frontends/pytorch/e2e_testing/torchscript/basic.py

```diff
@@ -101,3 +101,22 @@ def forward(self, x):
 @register_test_case(module_factory=lambda: ReluModule())
 def ReluModule_basic(module, tu: TestUtils):
     module.forward(tu.rand(4, 2) - 0.5)
+
+class BatchNormModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+        self.bn2d = torch.nn.BatchNorm2d(2)
```

Comment:
Since you implemented support for many cases, can you also add BatchNorm1d and BatchNorm3d to verify the extreme cases of NC and NCDHW?